examples/js/loaders/FBXLoader.js

```diff
@@ -730,7 +730,14 @@
 
 		if ( subNodes.LayerElementUV ) {
 
-			var uvInfo = getUVs( subNodes.LayerElementUV[ 0 ] );
+			var uvInfo = [];
+			var i = 0;
+			while ( subNodes.LayerElementUV[ i ] ){
```

Comment:
Any reason this is using a `while` instead of a `for` loop?

Reply:
Yeah, I could not find a way to iterate over the subNodes.LayerElementUV array. The object has a length property, but for some reason the value is 0. Also, using a for loop such as

for (var uvNode in subNodes.LayerElementUV ) {}

results in no outputs. 

Let me know if you have any other suggestions though.

Reply:
@takahirox any ideas?

Reply:
`subNodes.LayerElementUV` is just an object where the property names are numbers: 

```
LayerElementUV = { 
    1: {},
    2: {},
    3: {},
}
```

At least, that's the way it looks in @sneha-belkhale's example file. I'm not 100% sure we can assume that these will always be named like this, so it would be better to use the `for...in` pattern.

```
for (var key in subNodes.LayerElementUV) {
    uvInfo.push(getUVs( subNodes.LayerElementUV[ i ] ));
}
``` 

I've tested the above btw and it does work.